python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/torch_ods_gen.py

```diff
@@ -258,6 +258,8 @@ def emit_with_mutating_variants(key, **kwargs):
             "aten::cos : (Tensor) -> (Tensor)",
             "aten::atan : (Tensor) -> (Tensor)",
             "aten::atan2 : (Tensor, Tensor) -> (Tensor)",
+            "aten::asin : (Tensor) -> (Tensor)",
```

Comment:
You need to also commit the files that change after you run `build_tools/update_abstract_interp_lib.sh` (this is why CI fails)